packages/components/ng-ovh-actions-menu/src/directive.js

```diff
@@ -121,26 +119,32 @@ export default function() {
 
         // manage focus loop for current page
         visiblePage.on('keydown', (event) => {
+          const firstLinkElement = visiblePage
```

Comment:
Are we sure we always have a result ? (as we now take first array element)

Reply:
In fact here, the `first()[0]` is here to get the element targeted HTML content, of the JSON returned by `find` + `first`, which is required to work with `event.target.matches`.

The jQuery method `is()` is able to work directly with the JSON return by `find` and `first`.